application/uitests/org.openjdk.jmc.flightrecorder.uitest/src/test/java/org/openjdk/jmc/flightrecorder/uitest/JfrThreadsPageTest.java

```diff
@@ -84,6 +93,32 @@ public void after() {
 		}
 	};
 
+	boolean selected = false;
+	@Test
+	public void testTextCanvasSelection() throws InterruptedException, ExecutionException, TimeoutException {
+		openThreadsTable();
+		threadsTable.selectItems(0,0);
```

Comment:
`[..].selectItems(0,0);` -> `[..].selectItems(0, 0);`

Reply:
Addressed in https://github.com/aptmac/jmc/pull/49/commits/ae6f31c61c1e5c64445508e0770d105703d2d1d5